src/test/kotlin/model/BlackJackTest.kt

```diff
@@ -0,0 +1,15 @@
+package model
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+
+class BlackJackTest {
+    @Test
+    fun `deal 2 cards for each player at the start of the game`() {
+        val dealer = Dealer()
+        val players = listOf("A", "B", "C", "D", "E", "F").map { Player(it) }
+        BlackJack.initGame(players, dealer)
+        Assertions.assertTrue { players.all { it.getCardsNumber() == 2 } }
+        Assertions.assertTrue { dealer.getCardsNumber() == 2 }
+    }
```

Comment:
Could you check this test?
I don't think `BlackJack` has a method called `initGame`.

<img width="4320" height="2478" alt="image" src="https://github.com/user-attachments/assets/918a9ffd-d061-4520-a9fb-d95fd3b59bf0" />